ports/stm32f3/boards.c

```diff
@@ -190,7 +190,11 @@ void board_app_jump(void)
 uint8_t board_usb_get_serial(uint8_t serial_id[16])
 {
   uint8_t const len = 12;
-  memcpy(serial_id, STM32_UUID, len);
+  uint32_t word;
+  for (int i = 0; i < len; i=i+4) {
+    word = READ_REG(*(STM32_UUID+i));
```

Comment:
thanks for above confirmation, the expression is not correct `STM32_UUID+i` with i = 4n is not correct since STM32_UUID is already declared as uint32_t*. I will try to install gcc11 and try later.

Reply:
oh thats true. Good catch.